django-stubs/core/mail/message.pyi

```diff
@@ -60,15 +65,15 @@ class EmailMessage:
     def __init__(
         self,
         subject: str = ...,
-        body: Optional[str] = ...,
+        body: str = ...,
```

Comment:
I believe `body` can be `None`, example is here
https://github.com/django/django/blob/master/tests/mail/tests.py#L366